app-server/app/src/main/scala/scene/Scene.scala

```diff
@@ -0,0 +1,100 @@
+package com.azavea.rf.scene
+
+import com.azavea.rf.utils.Database
+import java.sql.Timestamp
+
+import com.azavea.rf.datamodel.latest.schema.tables._
+import com.azavea.rf.datamodel.latest.enums.JobStatus._
+import com.azavea.rf.datamodel.latest.enums.Visibility._
+
+import scala.concurrent.{Future, ExecutionContext}
+
+import scala.util.{Try, Success, Failure}
+
+/** Handles interaction between scenes and database */
+trait Scene {
+
+  implicit val database:Database
+  implicit val ec:ExecutionContext
+
+  import database.driver.api._
+
+  /** Insert a scene into the database
+    *
+    * @param scene ScenesRow
+    */
+  def insertScene(scene: ScenesRow): Future[Try[ScenesRow]] = {
+    database.db.run {
+      Scenes.forceInsert(scene).asTry
+    } map {
+      case Success(_) => Success(scene)
+      case Failure(e) => Failure(e)
```

Comment:
Not important here, but a handy trick for these match cases where you want to return the thing matched against:
```scala
SomeTry match {
  case s@Success(_) => s
  case f@Failure(e) => f
}
```